tests/test_resource_api.py

```diff
@@ -22,33 +22,51 @@
 user_header = {'Content-Type': "application/json"}
 user_header.update(get_auth_header(service_accounts['user']))
 
+rt_count = 0
 
-class TestApi(BaseAPITest, AssertJSONMixin, unittest.TestCase):
+
+def resource_type_name():
```

Comment:
Only in the interest of avoiding global variables, have you considered implementing `resource_type_name` and `rt_count` as class members?

Reply:
other than laziness, there is no good reason not, so I will.